src/utils/deep_freeze.js

```diff
@@ -26,6 +26,11 @@
  * only be used in DEV.
  */
 export default function deepFreezeAndThrowOnMutation(object) {
+    // Some objects in IE11 don't have a hasOwnProperty method so don't even bother trying to freeze them
+    if (window.navigator.userAgent.indexOf('Trident') !== -1) {
```

Comment:
hmmm, can we test for `window`, `navigator` and `userAgent` don't know if any of those is undefined with RN